models/User.js

```diff
@@ -1,47 +1,23 @@
-const User = (sequelize, DataTypes) => {
-  const user = sequelize.define('user', {
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('User', {
     username: {
       type: DataTypes.STRING,
-      unique: true,
-      validate: {
-        isUnique(value, next) {
-          const self = this;
-          user.find({ where: { username: value } })
-            .then((users) => {
-              if (users && self.id !== users.id) {
-                return next('Username already in use!');
-              }
-              return next();
-            })
-            .catch(err => next(err));
-        }
-      }
+      allowNull: false
+
```

Comment:
Add a unique constraint to the `username` as well.

Reply:
ok